inductiva/resources/machines.py

```diff
@@ -54,66 +58,76 @@ def start(self):
                 disk_size_gb=self.disk_size_gb,
                 zone=self.zone,
             )
-            # body.resource_pool_id = self.machine_group_id
             try:
                 logging.info("Creating a machine group. \
                              This may take a few minutes.")
-                api_instance.create_instance_group(body=body)
+                start_time = time()
+                api_instance.create_instance_group(body=instance_group_config)
```

Comment:
this is where you should get the id, if you need it.
If the id is still being created on the client, please leave a TODO so you can change it later to the server side.